agent/apps/ai_guardrails/app_code/models.py

```
@@ -0,0 +1,168 @@
+    severity = models.IntegerField(
+        default=50,
+        validators=[MinValueValidator(0), MaxValueValidator(100)],
+        verbose_name="شدت (0-100)",
+    )
+    is_active = models.BooleanField(default=True, verbose_name="فعال")
+    metadata = models.JSONField(default=dict, blank=True, verbose_name="متادیتا")
+
+    class Meta:
+        verbose_name = "قانون رد-فلگ"
+        verbose_name_plural = "قوانین رد-فلگ"
+        ordering = ["-severity", "-created_at"]
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)